drivers/census_tracking/app/models/census_tracking/worksheet.rb

```diff
@@ -0,0 +1,90 @@
+###
+# Copyright 2016 - 2021 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+module CensusTracking
+  class Worksheet
+    include ArelHelper
+
+    def initialize(filter)
+      @filter = filter
+    end
+
+    def projects
+      projects = GrdaWarehouse::Hud::Project.
+        preload(:organization).
+        viewable_by(@filter.user).
+        where(id: @filter.effective_project_ids)
+
+      projects.map { |project| [HUD.project_type(project.compute_project_type), project.organization.name, project.safe_project_name, project.id] }.
+        sort_by { |project| [project[0], project[1], project[2]] }
+    end
+
+    def clients_by_project(project_id, query)
+      scope = service_scope.in_project(project_id)
+      clients_by_population(query, scope: scope)
+    end
```

Comment:
This seems odd.  Why does clients_by_project return clients_by_population?  Maybe the outer one should be `client_data_for_project` or something similar.  It looks like it's returning data for one project, not organized by project.

Reply:
I think this is just over-DRYing... I'll push the `where` for the population up, and rename the population query parameter...